go/vt/vtgate/evalengine/vm.go

```diff
@@ -0,0 +1,61 @@
+package evalengine
+
+import (
+	"errors"
+
+	"vitess.io/vitess/go/sqltypes"
+)
+
+const MaxStack = 32
```

Comment:
Is this value based on anything or mostly pulled out of thin air? Dunno how deep the stack can get on evaluating common expressions?